tree-table/tests/unit/components/examples/TreeTableDefaultRow.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <tree-table class="table" :data="tableData" :columns="columns">
+  <tree-table class="table" :tableData="tableData" :columns="columns">
```

Comment:
We are using two different conventions for params ( I know, I'm the one responsible ;) ) We need to decide which we want to go for

Reply:
linter proposes the ones with dash inside e.g :table-data=''tableData". I must have missed this one